docs/philosophy.md

```diff
@@ -1,6 +1,12 @@
 The Philosophy of Scalar
 ========================
 
+> **Note: Scalar has moved!** It is now part of [the `microsoft/git` fork][microsoft-git]
+> and thus this document has [a new version in that repository][git-philosophy].
+
+[microsoft-git]: https://github.com/microsoft/git
+[git-philosophy]: https://github.com/microsoft/git/blob/HEAD/contrib/scalar/docs/philosophy.md
+
```

Comment:
Should we truncate this file here?  The above link will take them to the new version.  I'm thinking what follows here with get stale.